job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/BuildInDockerContext.groovy

```diff
@@ -14,6 +14,7 @@ class BuildInDockerContext extends AbstractContext {
     boolean forcePull
     String userGroup
     String startCommand = '/bin/cat'
+    String net = 'bridge'
```

Comment:
This is the default value. See
- https://github.com/jenkinsci/docker-custom-build-environment-plugin/blob/c3da4fa3e757e55ce78e3477d3e6f96b7ab865e2/src/main/resources/com/cloudbees/jenkins/plugins/docker_build_env/DockerBuildWrapper/config.jelly#L65